mne/event.py

```diff
@@ -885,3 +886,349 @@ def concatenate_events(events, first_samps, last_samps):
+                       'eeg': self.eegmax, 'eog': self.eogmax,
+                       'ecg': self.ecgmax}
+        self.reject = {k: float(v) for k, v in self.reject.iteritems()
+                       if float(v) > 0}
+        self.flat = {'grad': self.megmin, 'mag': self.magmin,
+                     'eeg': self.eegmin}
+        self.flat = {k: float(v) for k, v in self.flat.iteritems()
+                     if float(v) > 0}
+
+    def __repr__(self):
+        s = '<ElektaAverager | '
+        s += 'categories: %d ' % self.ncateg
+        cats_in_use = len(self._categories_in_use)
+        s += '(%d in use), ' % cats_in_use
+        s += 'events: %d ' % self.nevent
+        evs_in_use = len(self._events_in_use)
+        s += '(%d in use), ' % evs_in_use
+        s += 'stim source: %s' % self.stimsource
+        s += '\nCategories:\n'
+        for cat in self.categories:
+            s += '%d: "%s"\n' % (cat['index'], cat['comment'])
+        s += '>'
+        return s
+
+    def __getitem__(self, items):
+        return self._categories[items]
+
+    def _events_from_acq_pars(self):
+        """ Collect DACQ defined events into a dict. Events are keyed by number
+        starting from 1 (DACQ index of event). Each event is itself represented
+        by a dict containing the event definitions. """
+        events = {}
+        for evnum in range(1, self.ncateg + 1):
+            evnum_s = str(evnum).zfill(2)  # '01', '02' etc.
+            evdi = {}
+            for var in self.event_vars:
+                # name of DACQ variable, e.g. 'ERFeventNewBits01'
+                acq_key = 'ERFevent' + var + evnum_s
+                # corresponding dict key, e.g. 'newbits'
+                dict_key = var.lower()
+                val = self.acq_dict[acq_key]
+                # type convert numeric values
+                if dict_key in ['newbits', 'oldbits', 'newmask', 'oldmask']:
+                    val = int(val)
+                elif dict_key in ['delay']:
+                    val = float(val)
+                evdi[dict_key] = val
+                evdi['in_use'] = False  # __init__() will set this
+            evdi['index'] = evnum
+            events[evnum] = evdi
+        return events
+
+    def _categories_from_acq_pars(self):
+        """ Collects DACQ averaging categories into a dict. Categories
+        are keyed by the comment defined in DACQ. Each category is represented
+        a dict containing the category definitions. """
+        cats = {}
+        for catnum in [str(x).zfill(2) for x in range(1, self.nevent + 1)]:
+            catdi = {}
+            # read all category variables
+            for var in self.cat_vars:
+                acq_key = 'ERFcat' + var + catnum
+                class_key = var.lower()
+                val = self.acq_dict[acq_key]
+                catdi[class_key] = val
+            # some type conversions
+            catdi['display'] = True if catdi['display'] == '1' else False
```

Comment:
how about `catdi['display'] = (catdi['display'] == '1')`

Depending on the allowed values of catdi['display'], you may be able to do `bool(int(catdi['display']))`